CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Dates are in `yyyy-mm-dd`.
 
 ### Changed
 * Login Screen design
+* Changed Courses list for a new, modern looking design
```

Comment:
You don't need to have the `changed` again. 
The changelog is supposed to  be read as heading + entry: